skimage/filters/thresholding.py

```diff
@@ -727,14 +727,10 @@ def threshold_triangle(image, nbins=256):
     width /= norm
 
     # Maximize the length.
-    d = peak_height * arg_low_level - width * hist[arg_low_level]
```

Comment:
This is a constant across the x-coordinates of the triangle, so it has no effect on minimizing the length, i.e.,  d/dx f(x) + c  = d/dx f(x). 

To be honest, I was able to reproduce the equation for how the length varies over the triangle, but rather than making sure that that shift was the correct offset to measure the absolute length, when I realized it didn't affect the minimum I simply removed it.